Calibration/TkAlCaRecoProducers/plugins/AlcaBeamSpotHarvester.cc

```diff
@@ -155,7 +157,7 @@ void AlcaBeamSpotHarvester::endRun(const edm::Run& iRun, const edm::EventSetup&)
         //poolDbService->appendSinceTime<BeamSpotObjects>(aBeamSpot, poolDbService->currentTime(),"BeamSpotObjectsRcd");
 	poolDbService->writeOne<BeamSpotObjects>(aBeamSpot, thisIOV, outputrecordName_);
         if (dumpTxt_ && beamSpotOutputBase_ == "lumibased"){
-            beamspot::dumpBeamSpotTxt(outTxt, true, currentBS);
+            beamspot::dumpBeamSpotTxt(outFile, currentBS);
```

Comment:
I'm lost a bit in which set of changes provides a fix.
The PR description mentions a problem of overwriting a file.
This change, if the outTxt existed already will overwrite now, while it was going to append in the previous version.
this seems to contradict the PR description